File: firstclicks-server/src/main/java/com/project/firstclicks/dto/StudentCourseDTO.java
package com.project.firstclicks.dto;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class StudentCourseDTO {
	private Integer id;
	private CoursePublicDTO CourseEnrolled;
	private Integer studentId;
	private Boolean studentActive;
	private String studentReview;
	private Integer studentStars;
	private LocalDateTime studentJoinDate;
}

File: firstclicks-ui/src/services/models/student-course-dto.ts
/* tslint:disable */
/* eslint-disable */
import { CoursePublicDto } from '../models/course-public-dto';
export interface StudentCourseDto {
  courseEnrolled: CoursePublicDto;
  id?: number;
  studentActive?: boolean;
  studentId?: number;
  studentJoinDate?: string;
  studentReview?: string;
  studentStars?: number;
}
